### ISSUE TITLE
create logs for the entire das-query-engine function process including database access data and connection

### ISSUE THREAD

**Issue by levisingularity** [singnet/das-query-engine#51]

**Comment by levisingularity** [comment]
@andre-senna https://github.com/singnet/das-servless-functions/pull/24.

I've created a readme explaining how to visualize the logs created by the function.

**Comment by levisingularity** [comment]
Finding a way to visualize the logs was a headache because I practically found nothing, but I managed to succeed. Initially, I explored options like volume binding in the docker-compose.yml of faasd, but soon realized that it wasn't feasible. Faasd containers are separated from the gateway, making it impossible to establish a file system connection to make the binding work. Furthermore, I attempted to set up a volume through the template, only to discover that faasd doesn't support volume manipulation, severely limiting its container management capabilities.

Another approach I considered was using stdout. However, this posed issues with function return and log capturing. After trying various approaches, I finally found a solution. Even though containerd is just a container runtime, it's possible to access the container through ctr. Once this access was discovered, simply using a 'cat' command on the log file generated by Python's logging allowed access to the logs.

To streamline the process, I documented this solution in the serverless-function's readme, providing detailed instructions on how to execute the command to view and export the logs. It's essential to note that the logs remain available only while the container is running. If the container fails, all function logs are lost. However, if preserving these logs is necessary, it's possible to create a routine using crontab or a service in Linux to periodically export them to the host machine.
